src/yardstick/store/scan_result.py

```diff
@@ -77,7 +77,7 @@ def find(
     by_tool: str = "",
     by_time: str = "",
     by_description: str = "",
-    store_root: str = None,
+    store_root: Optional[str] = None,
```

Comment:
nit: I'd prefer if unrelated changes were in a separate PR.

Reply:
I highlighted that I was upgrading logging and type hints around store_root as found while integrating this into grype-db

Reply:
I saw that. I just wish it had all been done as a previous PR; that would have made it much easier to review both PRs.